### ISSUE TITLE
Import / Export buttons

### ISSUE THREAD

**Issue by nedredmond** [Khan/career-competencies#10]
Add import/export buttons that will overwrite context with JSON or download JSON file.

Add to "nav" component above `<Outlet />` so that it is present on every view as a header. Maybe rename from "nav", or import "nav" and "header" into new "root" component?

```
export const Root () => (
  <>
    <Header /> // with import/export
    <Outlet /> // main content
    <Nav /> // footer navigation
  </>
);
```

**Comment by nedredmond** [comment]
https://khanacademy.slack.com/archives/C060Y8JP8D7/p1698239895788839

**Comment by nedredmond** [comment]
@SonicScrewdriver hoping we can use some Phosphoricons here instead of the big ol words!

There should already be an action you can dispatch to replace the state... As far as exporting, I don't know how to do that yet! We can split this in 2 if you'd like

**Comment by nedredmond** [comment]
resolved by #37
